src/components/footer/index.tsx

```diff
@@ -82,7 +90,14 @@ function Footer({
 					let innerElement: ReactElement
 					if (item.type === 'link') {
 						innerElement = (
-							<a className={s.linkAction} href={item.href}>
+							<a
+								className={s.linkAction}
+								href={item.href}
+								aria-label={item.ariaLabel}
+								{...(item.external
+									? { target: '_blank', rel: 'noreferrer' }
+									: {})}
+							>
```

Comment:
I find that setting the `target` and `rel` props in an explicit way is easier to read than spreading an object alongside other explicitly set props. What do you think about updating this?

```suggestion
							<a
								className={s.linkAction}
								href={item.href}
								aria-label={item.ariaLabel}
								target={item.external ? '_blank' : undefined}
								rel={item.external ? 'noreferrer' : undefined}
							>
```

Reply:
Agreed it's easier to read - i think i'll have to add a lint-ignore (for target="_blank" + rel="noreferrer") but will give it a shot 👍